biscuit.gemspec

```diff
@@ -30,5 +22,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.9"
   spec.add_development_dependency "rake", "~> 10.0"
 
-  spec.add_runtime_dependency "rake", "~> 10.0"
+  spec.add_runtime_dependency "rake"
```

Comment:
👏🏼 I was just researching upgrading our rake gem in the orders repo and this was a major blocker. Big fan of this change.